### PR TITLE
added configurable cargo for http session

### PR DESCRIPTION
**Pull request by tommaso-borgato**
Added configurable cargo for http session via an optional query parameter; e.g.:
`http//<host>:8080/clusterbench/session?cargokb=40 `
means 40kb cargo (default is 4kb)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -151,4 +151,22 @@
...
+            <id>jboss-qa-releases</id>
+            <name>JBoss QA release repository</name>
+            <url>http://repository.engineering.redhat.com/nexus/content/repositories/jboss-qa-releases</url>
+        </repository>
+        <site>
+            <id>jboss-qa-sites</id>
+            <name>JBoss QA site repository</name>
+            <url>scp://dev01.mw.lab.eng.bos.redhat.com/qa/services/http/sites/</url>
+        </site>
+    </distributionManagement>
```
Comment (rhusar) [inline pom.xml]: These are not usable outside of one company thus I don't think this should be in this project. The repositories are supposed to be defined in `~/.m2/settings.xml`.

File: clusterbench-common/src/main/java/org/jboss/test/clusterbench/common/ClusterBenchConstants.java [hunk 2]
```diff
@@ -21,4 +21,6 @@
     String READONLY = "readonly";
 
     String INVALIDATE = "invalidate";
+
+    String CARGOKB = "cargokb";
```
Comment (rhusar) [inline clusterbench-common/src/main/java/org/jboss/test/clusterbench/common/ClusterBenchConstants.java]: Is there any reason to limit the granularity to KB (i.e. you cannot use 0.5 KB sessions should a test require it)? Maybe we could just do bytes.

Comment (tommaso-borgato) [reply clusterbench-common/src/main/java/org/jboss/test/clusterbench/common/ClusterBenchConstants.java]: using bytes looked overkill to me, anyway I can use bytes or even add some unit measures (e.g. byte kb mb)

Comment (tommaso-borgato) [reply clusterbench-common/src/main/java/org/jboss/test/clusterbench/common/ClusterBenchConstants.java]: @rhusar let me know if you want me to increase granularity and use bytes as unit measure

**Comment by rhusar** [comment]
TODO: docs, code cleanup (looks like certain calls can be simplified)

**Comment by rhusar** [comment]
Just a note for onlookers: IIRC this had been used to test state transfer logic.

**Pull request merged**